### DISCUSSION TITLE
Pages Build Failing Due to Billing

### DISCUSSION THREAD

**Discussion by annashansen** [community/community#32621] [Other Feature Feedback, Questions, & Ideas]
Hello! I'm trying to build a page using Pages but for some reason, the build fails because "recent account payments have failed or your spending limit needs to be increased." I did recently switch from the free plan to the student plan (which is technically pro), but both are free for me so I'm not sure what the issue is. I tried putting in my credit card information and increasing my spending limit, but the check still fails. 
![Screen Shot 2022-09-11 at 1.15.29 AM.png](https://github.zendesk.com/attachments/token/dJXKVPEiyqQG0QoIAkGqeWDQV/?name=Screen+Shot+2022-09-11+at+1.15.29+AM.png)

**Comment by radar-blip** [comment]
Not sure when you made the switch, but maybe it takes some time for you plan to be upgraded, 
or maybe even cc information.
Anyway, good luck!
